### PR TITLE
Support for the JSON.OBJKEYS command

### LINKED ISSUE #1820: Add support for the JSON.OBJKEYS command
Please refer to https://github.com/apache/kvrocks/issues/1805.

### PR DESCRIPTION
**Pull request by jihuayu**
close https://github.com/apache/kvrocks/issues/1820

Redis doc  [json.objkeys](https://redis.io/commands/json.objkeys/)

### CODE REVIEW

**Comment by git-hulk** [comment]
@jihuayu I think only Go tests is ok if most of them can be covered.

**Comment by jihuayu** [comment]
@git-hulk  I'm not quite understanding the difference in using between cpp test and go test in kvrocks. When should I add a cpp test, and when go test is OK?

**Comment by git-hulk** [comment]
> rstanding the difference in using between cpp test and go test in kvrocks. When should I add a cpp test, and when go test is

No, only Go is ok if the key path can be tested. And cpp test is used to cover the path which is hard to test through the redis protocol way.

File: src/commands/cmd_json.cc [hunk 1]
```diff
@@ -167,6 +167,29 @@ class CommandJsonType : public Commander {
...
 
+class CommandJsonObjkeys : public Commander {
+ public:
+  Status Execute(Server *srv, Connection *conn, std::string *output) override {
+    redis::Json json(srv->storage, conn->GetNamespace());
+
+    std::vector<std::optional<std::vector<std::string>>> results;
+
+    auto s = json.ObjKeys(args_[1], args_[2], results);
+    if (!s.ok()) return {Status::RedisExecErr, s.ToString()};
```
Comment (git-hulk) [inline src/commands/cmd_json.cc]: Should return null if key is non-exists, rest are good to me.

Comment (jihuayu) [reply src/commands/cmd_json.cc]: fixed

**Review by git-hulk — DISMISSED** [review]

**Review by git-hulk — APPROVED** [review]

**Pull request merged**